### ISSUE TITLE
optOptimizeLayout introducing edge profile inconsistencies

### ISSUE THREAD

**Issue by AndyAyersMS** [dotnet/runtime#48476]
See for instance https://github.com/dotnet/runtime/pull/48364#issuecomment-780129816

category:cq
theme:block-layout

**Comment by AndyAyersMS** [comment]
I think it's going to be impossible to get this right until we have persistent edge weights / successor likelihoods. So marking as future.

**Comment by AndyAyersMS** [comment]
@amanasifkhalid I assume we can close this now?

**Comment by amanasifkhalid** [comment]
> @amanasifkhalid I assume we can close this now?

Yes, thanks for tagging me on this -- looks like there might be other issues (i.e. #50419) we can close.
